common/lib/protocol-definitions/src/summary.ts

```diff
@@ -7,13 +7,21 @@ export type SummaryObject = ISummaryTree | ISummaryBlob | ISummaryHandle | ISumm
 
 export type SummaryTree = ISummaryTree | ISummaryHandle;
 
+/**
+ * @deprecated ISummaryAuthor has never been used anywhere and is unlikely to be used in the
```

Comment:
@tylerbutler should I also change the comment here to the generic "are deprecated and will be removed in a future release."?

Reply:
Yeah, I would.